docs/tutorials/enterprise_beginner/3_use_existing_blocks.md

```diff
@@ -51,7 +51,7 @@ blocks:
 
 The template makes use of an [image variation](../../guide/images.md) (line 10).
 It is the thumbnail of the Dog Breed image that will be displayed in the block.
-To configure this variation, open the `app/config/image_variations.yml` file and add the following code under the `image_variations` key:
+To configure this variation, open the `app/config/image_variations.yaml` file and add the following code under the `image_variations` key:
```

Comment:
File path is wrong. I'm not sure about this one but most likely it should be located in `config/packages/ezplatform.yaml`